projects/common/src/utilities/validators/form-validators.ts

```diff
@@ -0,0 +1,14 @@
+import { AbstractControl } from '@angular/forms';
+import { getStringsFromCommaSeparatedList } from '../formatters/string/string-formatter';
+import { areEmailAddressesValid } from './email-validator';
+
+export const validateCommaSeparatedEmails = (control: AbstractControl) => {
```

Comment:
I think `commaSeparatedEmailsValidator` is a better name for this validator.
IMO, we should namespace all the custom validators (at least the new ones) with something like
`CustomValidators.commaSeparatedEmails`.
Here it will be easier to do since we only have one validator for now. We can move others later to this.

Reply:
updated